recipe/test_openff_toolkit.py

```diff
@@ -13,6 +13,7 @@
 from openff.toolkit.topology import Molecule, Topology
 from openff.toolkit.typing.engines.smirnoff import ForceField
 offmol = Molecule.from_smiles('CCO')
+Molecule.from_rdkit(offmol.to_rdkit())
```

Comment:
I was marginally concerned about #1184 bringing in a new dep; in principle this or another step should fail when importing `RDKitToolkitWrapper` if an import fails